modules/ROOT/pages/op-tkn-clustertask-management.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// *  pipelines/op-tkn-reference.adoc
+
+[id="tkn-clustertask-management-commands_{context}"]
+= Clustertask management commands
```

Comment:
It is best practice and easier to maintain the same names for file names and ID/context.

Reply:
Ahh...I thought op-tkn should not be part of the id. It should just be title(heading) of the module. I will modify all the files.